.github/workflows/osx-release.yml

```diff
@@ -0,0 +1,150 @@
+#                <key>CFBundleTypeOSTypes</key>
+#                <array>
+#                  <string>fold</string>
+#                </array>
+#                <key>CFBundleTypeRole</key>
+#                <string>Viewer</string>
+#                <key>LSHandlerRank</key>
+#                <string>None</string>
+#              </dict>
+#              <dict>
+#                <key>CFBundleTypeIconFile</key>
+#                <string>Document</string>
+#                <key>CFBundleTypeName</key>
+#                <string>Document</string>
+#                <key>CFBundleTypeOSTypes</key>
+#                <array>
+#                  <string>****</string>
+#                </array>
+#                <key>CFBundleTypeRole</key>
+#                <string>Editor</string>
+#              </dict>
+#            </array>
+#            <key>CFBundleExecutable</key>
+#            <string>love</string>
+#            <key>CFBundleIconFile</key>
+#            <string>OS X AppIcon</string>
+#            <key>CFBundleIdentifier</key>
+#            <string>com.airthings.radon_run</string>
+#            <key>CFBundleInfoDictionaryVersion</key>
+#            <string>6.0</string>
+#            <key>CFBundleName</key>
+#            <string>radon_run</string>
+#            <key>CFBundlePackageType</key>
+#            <string>APPL</string>
+#            <key>CFBundleShortVersionString</key>
+#            <string>0.10.2</string>
+#            <key>CFBundleSignature</key>
+#            <string>LoVe</string>
+#            <key>CFBundleSupportedPlatforms</key>
+#            <array>
+#              <string>MacOSX</string>
+#            </array>
+#            <key>DTCompiler</key>
+#            <string>com.apple.compilers.llvm.clang.1_0</string>
+#            <key>DTPlatformBuild</key>
+#            <string>8B62</string>
+#            <key>DTPlatformVersion</key>
+#            <string>GM</string>
+#            <key>DTSDKBuild</key>
+#            <string>16B2649</string>
+#            <key>DTSDKName</key>
+#            <string>macosx10.12</string>
+#            <key>DTXcode</key>
+#            <string>0810</string>
+#            <key>DTXcodeBuild</key>
+#            <string>8B62</string>
+#            <key>LSApplicationCategoryType</key>
+#            <string>public.app-category.games</string>
+#            <key>NSHighResolutionCapable</key>
+#            <true/>
+#            <key>NSHumanReadableCopyright</key>
+#            <string>© 2020 Airthings Development Team</string>
+#            <key>NSPrincipalClass</key>
+#            <string>NSApplication</string>
+#            <key>UTExportedTypeDeclarations</key>
+#          </dict>
+#          </plist>
+#        EOT
+    - name: Package release
+      run: zip -r radon_run-macosx-x64.zip radon_run.app
+                
+    - name: Create Release
+      id: create_release
+      uses: actions/create-release@v1
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      with:
+        tag_name: ${{ github.ref }}
```

Comment:
this is probably wrong 🤦, need to pickup the tag so that the next release does not conflict with the first.